Example/Podfile

```diff
@@ -3,7 +3,7 @@ platform :ios, '11.0'
 
 target 'TrustWeb3Provider_Example' do
   pod 'TrustWeb3Provider', :path => '../'
-  #pod 'TrustWalletCore'
+  pod 'TrustWalletCore'
```

Comment:
can we just include models for Wallet Core?

Reply:
This is the example app to test, not the podspec itself

Reply:
correct, would it be possible to try just models?

Reply:
no, we need to test signing

Reply:
https://github.com/trustwallet/trust-web3-provider/blob/731666378ff9ca32c2729f1892c07795c6de6aa4/TrustWeb3Provider.podspec doesn't depend on wallet core

Reply:
got it, I was thinking to try test models pod :) looks good then